### PR TITLE
Move the dial pad button next to Explore button from Create Room context menu 

### PR DESCRIPTION
**Pull request by anoadragon453**
Fixes https://github.com/vector-im/element-web/issues/17652

This PR moves the dial pad from a context menu to sit right by the explore button, in line with the designs in the linked issue. The dial pad button is only shown when PSTN support is detected.

2525a95ee9e34ca1b415e9738c9110a6

fd77f36c50ff44caa9a5b39de93a2c01

Clicking it will bring up the dial pad. Best reviewed per-commit.

Signed-off-by: Andrew Morgan <andrewm@element.io>

### CODE REVIEW

File: src/components/views/rooms/RoomList.tsx [hunk 1]
```diff
@@ -98,44 +97,10 @@ interface ITagAesthetics {
     defaultHidden: boolean;
 }
 
-interface ITagAestheticsMap {
+const TAG_AESTHETICS: {
```
Comment (anoadragon453) [inline src/components/views/rooms/RoomList.tsx:100]: Removing the existing dial pad button involved removing a lot of the changes in https://github.com/matrix-org/matrix-react-sdk/pull/5523/files#diff-f58269671c69c23ef15a599c0ba04d5aa22016ab65b79832e84a76b9114777bcR215

However, I'm not entirely sure about the `TAG_AESTHETICS` bits and whether that was used to hide VoIP-related rooms, or simply used for the context menu. I've removed them, and there seems to be no adverse effects - but I'd like to double-check that those changes weren't necessary to hide PSTN rooms.

Comment (turt2live) [reply src/components/views/rooms/RoomList.tsx:100]: can we keep the interface definition? It makes the TS slightly less daunting to look at.

TagAesthetics are responsible for mostly what the `+` button does on a sublist, if anything, and naming for the default lists. It should be kept.

**Review by turt2live — COMMENTED** [review]
generally looks sane!

File: src/components/views/rooms/RoomList.tsx (hunk 1 above)
Comment (anoadragon453) [reply src/components/views/rooms/RoomList.tsx]: I've kept the interface definition, but removed all modifications of it - which I believe is correct?

Comment (turt2live) [reply src/components/views/rooms/RoomList.tsx]: I think that's fine - I didn't see anything particularly concerning about the removal.

**Review by turt2live — APPROVED** [review]
lgtm code-wise

**Review by gaelledel — APPROVED** [review]

**Pull request merged**